Packs/CortexAttackSurfaceManagement/Scripts/InferWhetherServiceIsDev/InferWhetherServiceIsDev.py

```diff
@@ -0,0 +1,149 @@
+                            or any([m == value for m in EXACT_DEV_MATCH])
+                            or any([m in value for m in PARTIAL_DEV_MATCH])):
+                        has_dev_indicator = True
+                    elif (any([m == value for m in EXACT_PROD_MATCH])
+                          or any([m in value for m in PARTIAL_PROD_MATCH])):
+                        has_prod_indicator = True
+
+    if has_dev_indicator and not has_prod_indicator:
+        return True
+    else:
+        return False
+
+
+def is_dev_according_to_classifications(classifications: List[str]) -> bool:
+    """
+    Returns whether any of the classification strings indicate that this service is
+    a development environment. The Xpanse ASM classification strings are a defined
+    vocabulary of facts that are derivable from the publicly observed service without
+    joining on any additional data.
+
+    Args:
+        classifications (List[str]): list of Xpanse ASM classification terms
+            (a defined vocabulary)
+            Example value for `classifications`:
+                ["RdpServer", "SelfSignedCertificate"]
+
+    Returns:
+        bool: whether there is an indication within `classifications` that
+            the described system is used for development
+    """
+    DEV_ENV_CLASSIFICATION = "DevelopmentEnvironment"
+    is_dev = (DEV_ENV_CLASSIFICATION in classifications)
+    return is_dev
+
+
+""" MAIN FUNCTION """
+
+
+def main():
+    """
+    Identifies whether the service is a "development" server. Development servers
+    have no external users and run no production workflows. These servers might be
+    named "dev", but they might also be named "qa", "pre-production", "user
+    acceptance testing", or use other non-production terms. This automation uses
+    both public data visible to anyone (`active_classifications` as derived by
+    Xpanse ASM) as well as checking internal data for AI-learned indicators of
+    development systems (`asm_tags` as derived from integrations with non-public
+    systems).
+
+    Args:
+        asm_tags (List[Dict[str, Any]]): list of key-value dictionaries;
+            each dictionary within the list must contain the keys "Key" and "Value";
+            the values are arbitrary
+            Example value for `observed_key_value_pairs`:
+                [{"Key": "env", "Source": "AWS", "Value": "dev"},
+                 {"Key": "Name", "Source": "AWS", "Value": "ssh-ec2-machine-name"}]
+        active_classifications (List[str]): list of Xpanse ASM classification terms
+            (a defined vocabulary)
+            Example value for `classifications`:
+                ["RdpServer", "SelfSignedCertificate"]
+
+    Returns:
+        bool: whether there is an indication within either `asm_tags` or
+            `active_classifications` that the described system is used for development
+    """
+    try:
+        args = demisto.args()
```

Comment:
please return CommandResults in the end of the main method and update the outputs of the script

Reply:
@adi88d , is this really necessary if we really only want to update a single field in our script?  I guess we could update context and then a separate playbook task, but that makes less sense to me.

Reply:
its necessary so the user can understand if the dev or not, please ad this block and update the `hr` and `context` value before:

`return_results(CommandResults(readable_output=hr, outputs=context)`